common/README.md

```diff
@@ -0,0 +1,5 @@
+# CLDR `common` directory
+
+This directory contains CLDR data corresponding to the release.
```

Comment:
This line "contains CLDR data corresponding to the release" is based on https://cldr.unicode.org/index/downloads#repository-organization

Likewise other new readme files should get descriptions based on that link